### ISSUE TITLE
Images contained in .json files

### ISSUE THREAD

**Issue by LukasMosser** [rgmyr/corebreakout#17]
I have a question regarding the assets.zip.

It looks like the json files are very large and after the "image_path" keyword, they seem to contain strange strings.
Could it be that these files also contain the images as well?

**Comment by rgmyr** [comment]
Good catch! I somehow failed to notice that. 

We may refactor `PolygonDataset` to use that data instead of `jpeg`s at some point, or release new data with that issue fixed, but for now I've just pushed d2a6ea9 which adds `scripts/prune_imageData.py` and some associated notes in the docs/README.

**Comment by LukasMosser** [comment]
I think keeping images as jpegs and the labels as jsons would make it easier to version the dataset. Simple diffs on the jsons are possible and makes it overall more lightweight.

**Comment by rgmyr** [comment]
Yeah actually I agree, and I don't think the ability to browse the images in a plain old image viewer is something we'd want to get rid of.
